contracts/FungibleTokenSwitchboard.cdc

```diff
@@ -0,0 +1,151 @@
+import FungibleToken from "./FungibleToken.cdc"
+
+/// FungibleTokenSwitchboard
+///
+/// The contract that allows an account to receive payments in multiple fungible
+/// tokens using a single {FungibleToken.Receiver}
+///
+pub contract FungibleTokenSwitchboard {
+    
+    // Storage and Public Paths
+    pub let SwitchboardStoragePath: StoragePath
+    pub let SwitchboardPublicPath: PublicPath
+    pub let SwitchboardReceiverPublicPath: PublicPath
+    
+    /// FungibleTokenSwitchboardInitialized
+    ///
+    /// The event emitted when the contract is created
+    ///
+    pub event FungibleTokenSwitchboardInitialized()
+    
+    /// SwitchboardInitialized
+    ///
+    /// The event that is emited when a new Switchboard resource is created
+    ///
+    pub event SwitchboardInitialized(switchboardResourceID: UInt64)
+
+    /// VaultCapacityAdded
+    ///
+    /// The event that is emitted when a new vault capacity is added to a
+    /// switchboard resource
+    ///
+    pub event VaultCapabilityAdded(type: Type)
```

Comment:
For all the events, I would think about what additional information might be useful to include. A good rule of thumb is to try to include any information that might be useful in events just in case people need them. For example, might be good to add an `address` field here and in the other ones. there may be more I'm not thinking of

Reply:
Great, I believe I have to learn about the real use case of consuming events, so far I have been coding them like mirroring for the examples but without having in mind how they will be used.